src/Organizer/DefaultOrganizerEditingService.php

```diff
@@ -150,4 +151,11 @@ public function delete(string $id): void
             new DeleteOrganizer($id)
         );
     }
+
+    public function removeAddress(string $organizerId): void
```

Comment:
Okay for now, but I'd suggest you and @jdrieghe try to remove these services in the long run. In most cases they don't do anything extra aside from dispatching a command, and in the other rare cases they do something extra it might be able to move it to the command handler instead. In the case of the `create` method, a `CreateOrganizer` command could be added. But this is not required for this PR, just a general comment for a possible future improvement.